internal/compiler/typeloader.rs

```diff
@@ -1582,6 +1582,7 @@ impl TypeLoader {
             };
             crate::fileaccess::load_file(path.as_path())
                 .map(|virtual_file| (virtual_file.canon_path, virtual_file.builtin_contents))
+                .or_else(|| Some((path, None)))
```

Comment:
Ah, with this it'll be delegated to the loader callback?

Reply:
Contrary to the name, `fileaccess::load_file` only load builtin files.
But it also try to canonicalize the path, and that doesn't work with wasm.
If the path doesn't exist, we can still return something here because we don't want to go  further in include path anyway. But i'll double check it still give the right error for non-existing files in libraries.